backend/src/main/java/com/empire/NationData.java

```diff
@@ -1,12 +1,36 @@
 package com.empire;
 
+import com.google.gson.annotations.SerializedName;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.security.SecureRandom;
 
 class NationData {
+	enum Tag {	
+		@SerializedName("Bloodthirsty") BLOODTHIRSTY,
+		@SerializedName("Coast-Dwelling") COAST_DWELLING,
+		@SerializedName("Defensive") DEFENSIVE,
+		@SerializedName("Disciplined") DISCIPLINED,
+		@SerializedName("Evangelical") EVANGELICAL,
+		@SerializedName("Heroic") HEROIC,
+		@SerializedName("Holy") HOLY,
+		@SerializedName("Imperialistic") IMPERIALISTIC,
+		@SerializedName("Industrial") INDUSTRIAL,
+		@SerializedName("Mercantile") MERCANTILE,
+		@SerializedName("Mystical") MYSTICAL,
+		@SerializedName("Nomadic") NOMADIC,
+		@SerializedName("Patriotic") PATRIOTIC,
+		@SerializedName("Rebellious") REBELLIOUS,
+		@SerializedName("Republican") REPUBLICAN,
+		@SerializedName("Ruined") RUINED,
+		@SerializedName("Seafaring") SEAFARING,
+		@SerializedName("Ship-Building") SHIP_BUILDING,
+		@SerializedName("Stoic") STOIC,
+		@SerializedName("War-like") WARLIKE;
```

Comment:
I don't seen any misspellings but the hyphenated words are inconsistent as far as capitalization after the hyphen.  The same applies to one or two army tags as well.  Whether lexically correct or not, I think it might be wiser to pick one convention and stick to it.  In this particular case I believe warlike can actually be a single word though.

Reply:
Ah yes - so in this case, I just wanted to check that I didn't accidentally change them from the previous values (because that would have to be updated in the FE half too). Following-up with a change that alters them in both seems like a good idea.